simulation/core/sim_dataset.py

```diff
@@ -290,7 +290,7 @@ def __init__(self,
         # Check that cache limit is possible.
         if cache_limit:
             self.cache_limit = normalize_bytes(cache_limit)
-            min_cache_usage = sum((stream.get_index_size() for stream in streams))
+            min_cache_usage = sum((stream.got_index_size for stream in streams))
```

Comment:
nit: why not just `stream.index_size`?

Reply:
https://github.com/mosaicml/streaming/pull/585/commits/eb64e40ea18ea506160adeb4b39c1f0213470b08